terraso_backend/apps/project_management/graphql/site_notes.py

```diff
@@ -0,0 +1,75 @@
+# Copyright © 2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+import graphene
+from graphene_django import DjangoObjectType
+
+from apps.project_management.models.site_notes import SiteNote
+from apps.project_management.models.sites import Site
+
+
+class SiteNoteNode(DjangoObjectType):
+    class Meta:
+        model = SiteNote
+        fields = "__all__"
+        interfaces = (graphene.relay.Node,)
+
+
+class SiteNoteAddMutation(graphene.Mutation):
```

Comment:
Issue: I think the mutations in this files are not authenticating the users, you can extend these classes from the classes in https://github.com/techmatters/terraso-backend/blob/main/terraso_backend/apps/graphql/schema/commons.py#L182 (BaseWriteMutation, BaseAuthenticatedMutation) to add that automatically